update-official.sh

```
@@ -0,0 +1,11 @@
+cd /workspaces/nuclei/nuclei-templates/ 2>/dev/null
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)